<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Static8</class>
 <widget class="QWidget" name="Static8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Static Gen 8</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxRNGInfo">
     <property name="title">
      <string>RNG Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QLabel" name="labelMaxAdvances">
        <property name="text">
         <string>Max Advances</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelInitialAdvances">
        <property name="text">
         <string>Initial Advances</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="TextBox" name="textBoxMaxAdvances">
        <property name="text">
         <string notr="true">100000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelLead">
        <property name="text">
         <string>Lead</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSeed0">
        <property name="text">
         <string>Seed 0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TextBox" name="textBoxSeed1"/>
      </item>
      <item row="0" column="1">
       <widget class="ComboMenu" name="comboMenuLead"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSeed1">
        <property name="text">
         <string>Seed 1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="TextBox" name="textBoxInitialAdvances">
        <property name="text">
         <string notr="true">0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="TextBox" name="textBoxSeed0"/>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="TextBox" name="textBoxDelay"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelDelay">
        <property name="text">
         <string>Delay</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBoxFilters">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="Filter" name="filter" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue"/>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="labelProfileGameValue"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue"/>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="TableView" name="tableView"/>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCategory">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxCategory">
        <item>
         <property name="text">
          <string>Starters</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gifts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fossils</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stationary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Roamers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Legends</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ramanas Park (Pure Space)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ramanas Park (Strange Space)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mythics</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPokemon">
        <property name="text">
         <string>Pokemon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ComboBox" name="comboBoxPokemon"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelLevel">
        <property name="text">
         <string>Level</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxLevel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ability</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ComboBox" name="comboBoxAbility">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">H</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">0/1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelShiny">
        <property name="text">
         <string>Shiny</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ComboBox" name="comboBoxShiny">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelIVCount">
        <property name="text">
         <string>IV Count</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spinBoxIVCount">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Form/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboMenu</class>
   <extends>QToolButton</extends>
   <header>Form/Controls/ComboMenu.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>comboMenuLead</tabstop>
  <tabstop>textBoxSeed0</tabstop>
  <tabstop>textBoxSeed1</tabstop>
  <tabstop>textBoxInitialAdvances</tabstop>
  <tabstop>textBoxMaxAdvances</tabstop>
  <tabstop>textBoxDelay</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>comboBoxCategory</tabstop>
  <tabstop>comboBoxPokemon</tabstop>
  <tabstop>spinBoxLevel</tabstop>
  <tabstop>comboBoxAbility</tabstop>
  <tabstop>comboBoxShiny</tabstop>
  <tabstop>spinBoxIVCount</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
